dialogue-core/src/main/java/com/palantir/dialogue/core/FixedLimitedChannel.java

```diff
@@ -82,4 +82,16 @@ private void logExhaustion(Endpoint endpoint) {
                     SafeArg.of("totalPermits", totalPermits));
         }
     }
+
+    @Override
+    public String toString() {
+        return "FixedLimitedChannel{"
+                + "delegate="
+                + delegate
+                + ", usedPermits="
+                + usedPermits
```

Comment:
Heh, this could be confusing because toString isn't evaluated when limits are, but I'm all for more data.